src/game/ArenaTeam.cpp

```diff
@@ -530,8 +530,8 @@ uint32 ArenaTeam::GetPoints(uint32 MemberRating)
     else
         points = 1511.26f / (1.0f + 1639.28f * exp(-0.00412f * (float)rating));
     
-    // Increase Arena Points due to shorter Seasons and smaller Battlegroup
-    points = points * 2.0;
+    // Rate.Arena.Points
+    points *= RATE_ARENA_POINTS;
```

Comment:
points *=  sWorld.getRate(RATE_ARENA_POINTS);

I wonder that there was no error :>